app/assets/javascripts/host_edit.js

```diff
@@ -146,13 +146,10 @@ function update_capabilities(capabilities) {
 var stop_pooling;
 
 function submit_with_all_params() {
-  var url = $('form').attr('action');
-  if (url.match('hostgroups')) {
-    resource = 'hostgroup';
-  } else {
-    resource = 'host';
-  }
-  capitalized_resource = resource[0].toUpperCase + resource.slice(1);
+  var url = $('form').attr('action'),
+  resource = 'host',
+  page_state_title = page_title_by_resource(resource),
```

Comment:
As this is used only once, I would prefer using the method right away there (not storing it in variable at all)